repositories/RoutingRepository.js

```diff
@@ -150,6 +157,16 @@ async function updateRoutingRuleSet({ id, else: destination }) {
   return routingRuleSet;
 }
 
+async function getAnswerTypeByConditionId(id, ctx) {
```

Comment:
I'm not sure I like the repository getting the `ctx`. Can't we just import the other repository?